base/dcp_feed.go

```diff
@@ -156,14 +153,14 @@ func (r *DCPReceiver) OnError(err error) {
 func dcpKeyFilter(key []byte) bool {
 
 	// If it's not a _sync doc, process
-	if !bytes.HasPrefix(key, []byte("_sync")) {
+	if !bytes.HasPrefix(key, []byte(SyncXattrName)) {
 		return true
 	}
 
 	// User, role and unused sequence markers should be processed
-	if bytes.HasPrefix(key, []byte("_sync:unused")) ||
-		bytes.HasPrefix(key, []byte("_sync:user:")) ||
-		bytes.HasPrefix(key, []byte("_sync:role:")) {
+	if bytes.HasPrefix(key, []byte(UnusedPrefixNoColon)) ||
```

Comment:
I think the intention of the code here was to check for `UnusedSeqPrefix`. Better to change it than introduce a new constant.